rules/macos/privilege_escalation_local_user_added_to_admin.toml

```diff
@@ -0,0 +1,45 @@
+[metadata]
+creation_date = "2020/01/05"
+maturity = "production"
+updated_date = "2020/01/05"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies attempts to add an account to the admin group via the command line. This could be an indication of privilege escalation activity.
```

Comment:
```suggestion
Identifies attempts to add an account to the admin group via the command line. This could be an indication of privilege
escalation activity.
```